src/platform/site-wide/announcements/config/index.js

```diff
@@ -1,10 +1,9 @@
-import DashboardIntro from '../components/DashboardIntro';
+import RecommendationsIntro from '../components/RecommendationsIntro';
```

Comment:
Should this be named something "preferences"-related?

Reply:
Preferences seems pretty general for a side-wide modal. But maybe something like FindVABenefitsIntro would be better.